sympy/polys/rings.py

```diff
@@ -2531,15 +2531,140 @@ def compose(f, x, a=None):
 
         return poly
 
+
+    def coeff_wrt(self, x, dg):
+        """
+        Coefficient of ``f`` with respect to ``x**deg``.
```

Comment:
The parameter is called `dg` but the docstring refers to `deg`. These should be made consistent. I suggest using `deg` as the parameter name.

Reply:
Also there is no parameter `f`.